Ipv6 network error on setting up new ceph dev environment

### Issue:

The following network error is generated when `docker compose up` command is ran:

```
Attaching to ceph
ceph             | npm ERR! code ERR_SOCKET_TIMEOUT
ceph             | npm ERR! errno ERR_SOCKET_TIMEOUT
ceph             | npm ERR! network request to https://registry.npmjs.org/zwitch/-/zwitch-1.0.5.tgz failed, reason: Socket timeout
ceph             | npm ERR! network This is a problem related to network connectivity.
ceph             | npm ERR! network In most cases you are behind a proxy or have bad network settings.
ceph             | npm ERR! network 
ceph             | npm ERR! network If you are behind a proxy, please make sure that the
ceph             | npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
ceph             | 
ceph             | npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2024-01-10T13_05_46_716Z-debug-0.log
```

### Cause/Reason:

Doing a ping and curl from inside the container tells that its a problem associated with IPv6

```
[root@ceph ceph]# ping registry.npmjs.org
PING registry.npmjs.org(2606:4700:8393:80e2:e439:411:6810:23 (2606:4700:8393:80e2:e439:411:6810:23)) 56 data bytes

^C
--- registry.npmjs.org ping statistics ---
144 packets transmitted, 0 received, 100% packet loss, time 146459ms
```
The presence of IPv6 address 2404:6800:4009:825::200e, suggests that the network communication is being attempted over IPv6. 

Also, the docker-compose.yaml is enabling use of ipv6:

https://github.com/rhcs-dashboard/ceph-dev/blob/7a72a850ed023110c5dd22528b2f4a9fd5986bb4/docker-compose.yml#L178

### Possible Resolution:

Configuring the docker to allow containers to use ipv6 addresses:

The [official documentaion](https://docs.docker.com/config/daemon/ipv6/) of docker-compose suggests following while using ipv6 containers:

> Before you can use IPv6 in Docker containers, you need to enable IPv6 support in the Docker daemon. Afterward, you can choose to use either IPv4 or IPv6 (or both) with any container or network.

**Steps to configure:**

1. Edit the Docker daemon configuration file, located at `/etc/docker/daemon.json`. Configure the following parameters:
```
{
  "experimental": true,
  "ip6tables": true
}
```

ip6tables enables additional IPv6 packet filter rules, providing network isolation and port mapping. This parameter requires experimental to be set to true.

_Note: you might need to create a new `/etc/docker/daemon.json` if its not present_

2. Save the configuration file.

3. Restart the Docker daemon for your changes to take effect.

 ```
 sudo systemctl restart docker
 ```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -175,7 +175,7 @@ volumes:
 networks:
     default:
         driver: bridge
-        enable_ipv6: true
+        enable_ipv6: false
         ipam:
             driver: default
             config:

